package.json

```diff
@@ -23,7 +23,8 @@
     "ecosystem:phonegap"
   ],
   "engines": {
-    "cordova": ">=3.0.0"
+    "cordova": ">=3.0.0",
+    "cordova-android": ">=6.3.0"
```

Comment:
@jcesarmobile can you set it so that version 1.3.0 and higher depend on cordova-android >=6.3.0? Then I'll release a 1.3.0 version.

Reply:
Sure, didn't want to add versions as wasn't sure if you were going to do a minor or major release.